encrypted-media-respec.html

```diff
@@ -2195,7 +2207,7 @@
         </p>
         <p>The CDM MUST ensure that data for a given session is only present in one active unclosed session in any <a def-id="document-concept"></a>.
           In other words, <a def-id="load"></a> MUST fail when there is already a <a>MediaKeySession</a> representing the session specified by the <var>sessionId</var> parameter, either because the object that created it via <a def-id="generateRequest"></a> is still active or it has been loaded into another object via <a def-id="load"></a>.
-          A session MAY only be loaded again after the <a def-id="session-close-algorithm"></a> algorithm has not been run on the object representing it.
+          A session MAY only be loaded again after the <a def-id="session-closed-algorithm"></a> algorithm has not been run on the object representing it.
```

Comment:
I think we will need separate definitions for _closed session_ and _closed `MediaKeySession`_. The latter implies the former, but not the other way around, first because the session is closed just before the `MediaKeySession` in the normal case and second because `MediaKeySession` can be destroyed without being closed, but the _session_ is still closed in this case.